modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -1542,6 +1558,42 @@ public void unloadTableResourcesFromZoneReplica(ZonePartitionId zonePartitionId,
         zoneResourcesManager.removeTableResources(zonePartitionId, tableId);
     }
 
+    /**
+     * Restarts the zone's partition including the replica and raft node.
+     *
+     * @param zonePartitionId Zone's partition that needs to be restarted.
+     * @param revision Metastore revision.
```

Comment:
Should `assignmentsTimestamp` be mentioned in a `@param` as other parameters are mentioned?

Reply:
done